### PR TITLE
fix(emailpro): display error message for spaces in password

### PR DESCRIPTION
**Pull request by frenautvh**
See : DTRUN-3225

### CODE REVIEW

File: packages/manager/modules/emailpro/src/account/add/emailpro-account-add.controller.js [hunk 1]
```diff
@@ -29,6 +30,12 @@ export default /* @ngInject */ ($scope, $stateParams, $translate, EmailPro, Emai
         $scope.differentPasswordFlag = true;
       }
 
+      if (selectedAccount.password.indexOf(' ') >= 0) {
```
Comment (FredericEspiau) [inline packages/manager/modules/emailpro/src/account/add/emailpro-account-add.controller.js]:
```suggestion
     $scope.containsSpace = _.includes(selectedAccount.password, ' ');
```
:)

File: packages/manager/modules/emailpro/src/account/update/emailpro-account-update.controller.js [hunk 2]
```diff
@@ -134,6 +136,12 @@ export default class EmailProUpdateAccountCtrl {
         this.newAccountOptions.minPasswordLength,
       );
 
+      if (selectedAccount.password.indexOf(' ') >= 0) {
```
Comment (FredericEspiau) [inline packages/manager/modules/emailpro/src/account/update/emailpro-account-update.controller.js]:
```suggestion
     this.containsSpace = _.includes(selectedAccount.password, ' ');
```

**Comment by cbourgois** [comment]
https://github.com/ovh-ux/ovh-module-emailpro is not yet deprecated, so changes should by made in specific repository.
When merged, we should update the subtree of module emailpro in this repository.

**Comment by frenautvh** [comment]
https://github.com/ovh-ux/ovh-module-emailpro/pull/241

**Pull request closed without merging**